godet.go

```diff
@@ -420,6 +427,35 @@ func (remote *RemoteDebugger) Reload() error {
 	return err
 }
 
+// CaptureScreenshot takes a screenshot, uses "png" as default format.
+func (remote *RemoteDebugger) CaptureScreenshot(format string, quality int, fromSurface bool) ([]byte, error) {
```

Comment:
Ok, but I am not too fond of passing all parameters when defaults would be sufficient. I am merging this but I am thinking of replacing input parameters with functional options (https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis).

Also, I would prefer if you submit pull requests for new commands one command at a time (so I can quickly pick the "easy" ones and we can continue the discussions on the others.